testsuite/features/build_validation/core/allcli_sanity.feature

```diff
@@ -184,6 +184,27 @@ Feature: Sanity checks
     And "sle15sp3_ssh_minion" should communicate with the server using public interface
     And the clock from "sle15sp3_ssh_minion" should be exact
 
+@sle15sp4_client
```

Comment:
We should definitely convert this feature in a template, as we do for smoke tests.
So we don't need to care about it, when we include new clients.

Reply:
But ok for this PR.

Reply:
why not - but not here